### PR TITLE
Refresh: update Edge tutorial and delete 14 day trial (again)

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
- Deletes 14 day trial tutorial
- Updates Edge tutorial for deletion script and UI
- Updates Edge tutorial for ingress nginx and cert-manager scripts

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/edge.rst [hunk 1]
```diff
@@ -128,6 +128,36 @@ above. If one or more prerequisites fail, the script will notify you of this,
...
+Ingress nginx
+'''''''''''''
+
+For installing nginx, we recommend reading the `installation instructions`_ on
+the nginx website. A sample command, depending on the Kubernetes provider you
+are using, might be:
+
+.. code-block:: console
+
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.34.1/deploy/static/provider/do/deploy.yaml
```
Comment (SStorm) [inline docs/edge.rst]:
```suggestion
    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.46.0/deploy/static/provider/cloud/deploy.yaml
```

I suggest we use the one for Azure as the Digital Ocean one has some issues :)

File: docs/edge.rst [hunk 2]
```diff
@@ -248,8 +278,28 @@ Select your desired plan and proceed to the :ref:`configuration wizard
...
 
+.. _edge-delete-region:
+
+Delete a custom region
+======================
+
+In order to delete a custom region, click the trashcan icon at the bottom right
+of the custom region panel. A confirmation screen will appear warning that
+deletion of a custom region disables access to CrateDB Cloud for that region.
+To confirm deletion of the custom region, enter "LOCAL" into the form.
```
Comment (SStorm) [inline docs/edge.rst]:
```suggestion
To confirm deletion of the custom region, enter the name of the region into the form.
```

"LOCAL" just so happens to be the name of my region :)

Comment (matthijskrul) [reply docs/edge.rst]: Ha, that is funny. Shoulda thought of that 🤔

**Review by Taliik — APPROVED** [review]
lgtm

**Review by juanpardo — APPROVED** [review]

**Pull request merged**